extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/MessageBundleProcessor.java

```diff
@@ -236,11 +237,17 @@ AdditionalBeanBuildItem beans() {
     void initBundleContext(RecorderContext context, MessageBundleRecorder recorder,
             List<MessageBundleMethodBuildItem> messageBundleMethods,
             List<MessageBundleBuildItem> bundles,
-            BuildProducer<SyntheticBeanBuildItem> syntheticBeans) {
+            BuildProducer<SyntheticBeanBuildItem> syntheticBeans) throws BuildException {
 
         Map<String, Map<String, Class<?>>> bundleInterfaces = new HashMap<>();
         for (MessageBundleBuildItem bundle : bundles) {
-            Class<?> bundleClass = context.classProxy(bundle.getDefaultBundleInterface().toString());
+            final Class<?> bundleClass;
+            try {
+                bundleClass = Class.forName(bundle.getDefaultBundleInterface().toString(), true,
+                        Thread.currentThread().getContextClassLoader());
+            } catch (ClassNotFoundException e) {
+                throw new BuildException(e, Collections.emptyList());
```

Comment:
I would just throw the CNFE, I don't think wrapping it adds anything

Reply:
Sounds fine. I'll update it accordingly.

Reply:
Done. PR updated.